app/scripts/angular-apimock.js

```diff
@@ -77,38 +99,31 @@ angular.module('apiMock', [])
 			return keys;
 		}
 
-		// Taken from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
+		// Modified from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
 		function serializeValue(v) {
-			if (angular.isObject(v)) {
-				return angular.isDate(v) ? v.toISOString() : angular.toJson(v);
+			if (angular.isDate(v)) {
+				return toISOString(v);
 			}
+
 			return v;
 		}
 
-		// Taken from Angular 1.4.x: https://github.com/angular/angular.js/blob/720012eab6fef5e075a1d6876dd2e508c8e95b73/src/ngResource/resource.js#L405
-		function encodeUriQuery(val, pctEncodeSpaces) {
+		// Modified from Angular 1.4.x: https://github.com/angular/angular.js/blob/720012eab6fef5e075a1d6876dd2e508c8e95b73/src/ngResource/resource.js#L405
+		function encodeUriQuery(val) {
 			return encodeURIComponent(val).
 			replace(/%40/gi, '@').
 			replace(/%3A/gi, ':').
 			replace(/%24/g, '$').
 			replace(/%2C/gi, ',').
-			replace(/%20/g, (pctEncodeSpaces ? '%20' : '+'));
+			replace(/%20/g, '+');
```

Comment:
Thx ^^ the parameter wasn't used anyway. Just cargo cult code that came along when copying the stuff from the Angular source code.